README.adoc

```diff
@@ -30,14 +29,15 @@ Dependencies
 Building
 --------
 
-Make sure all dependencies are installed. Then run this.
+Make sure all dependencies are installed. 
 
+Run:
```

Comment:
```suggestion
Run:

```
As `Run:` has 4 letters and you use `----` in the next line this will be recognized as a heading and will inverse code blocks/normal text until the end of the file (check your [branch](https://github.com/Qazzquimby/kiwmi) to see this in effect). To prevent this, insert an empty line.